### PR TITLE
Add EswEventNameAttribute

### PR DESCRIPTION
**Pull request by spoland**
This commit adds the EswEventNameAttribute and tests.

### CODE REVIEW

File: src/Eshopworld.Core/EswEventNameAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Eshopworld.Core
+{
+    /// <summary>
+    /// Describes an Esw Event Name Attribute
```
Comment (VaclavK) [inline src/Eshopworld.Core/EswEventNameAttribute.cs]: we should probably expand on this

Comment (RuaidhriMcDonnell) [inline src/Eshopworld.Core/EswEventNameAttribute.cs]: You should describe the purpose of this attribute, so people know what its for + when to use it

Comment (spoland) [reply src/Eshopworld.Core/EswEventNameAttribute.cs]: Comments updated - should be better now! 😄

Comment (spoland) [reply src/Eshopworld.Core/EswEventNameAttribute.cs]: Updated 👍

**Comment by dguerin** [comment]
@VaclavK could you take a look and see why the PR check failed?

**Comment by VaclavK** [comment]
@dguerin sonar steps in pipeline are messed up
References service endpoint. PRs from repository forks are not allowed to access secrets in the pipeline. For more information see https://go.microsoft.com/fwlink/?linkid=862029

**Comment by VaclavK** [comment]
@dguerin @spoland 

disabled sonar steps for now

**Review by dguerin — APPROVED** [review]
LGTM

**Comment by dguerin** [comment]
@VaclavK there is still that annoying problem of not being able to resubmit the PR for PR Checkout without a code change. Code looks good to me so I think it can be merged in as is.

**Comment by VaclavK** [comment]
> 
> 
> @VaclavK there is still that annoying problem of not being able to resubmit the PR for PR Checkout without a code change. Code looks good to me so I think it can be merged in as is.

yeah it is annoying one indeed

**Comment by VaclavK** [comment]
@RuaidhriMcDonnell you happy with the doco change on attribute?

**Review by RuaidhriMcDonnell — APPROVED** [review]

**Comment by dguerin** [comment]
@spoland there is one thing we forgot here, you need to bump the semver version number on the .csprog so that it creates a new version in the nuget feed. it won't at the moment as it will not overright existing version which has been released

**Pull request merged**